app/components/App/index.js

```diff
@@ -29,6 +30,7 @@ class App extends Component {
           <Route exact path='/create-password' component={CreateAccount} />
           <Route exact path='/restore-account' component={RestoreAccount} />
           <Route exact path='/account' component={Account} />
+          <Route exact path='/whitelisted-dapps' component={WhitelistedDApps} />
```

Comment:
I recommend you to use ```react-loadable``` and make the routes async. At least those ones which are not the entry point.

When you update your app to webpack 4, by default your app will be splitted in chunks and the app will load much faster (you can achieve the same in webpack 3 using CommonsChunk Plugin)